_bep_collection/bep025.md

```diff
@@ -1,4 +1,4 @@
 ---
 redirect_to:
-  - https://docs.google.com/document/d/1chZv7vAPE-ebPDxMktfI9i1OkLNR2FELIfpVYsaZPr4/
----
\ No newline at end of file
+  - https://docs.google.com/document/d/1xeiFACAnK_X64kL6XoXo8vpZ9Bn6B5Xd3j59Le1clRE/edit?usp=sharing
```

Comment:
why are you changing the Google Docs?

Reply:
We have made many modifications so we have done the document again.

Reply:
Mmh I don't think that's a very good idea because now you have two drawbacks:

1. the comment and revision history is lost in your new document
1. people who previously knew the URL of your document are now lost and may confuse both documents

But I guess that change has already been made, so let's salvage what we can:

you should add a big disclaimer to the old document, pointing to the new one and laying out the reasoning for why there is a new document.

the old document should be made "read only" (but do NOT delete it, in order to preserve the file history).

Reply:
We have put a warning in the old document and modified it to "read-only". 
Thanks for everything

Reply:
that looks good :+1: